mri/operators/fourier/non_cartesian.py

```diff
@@ -378,9 +387,134 @@ def adj_op(self, x):
         return img * np.sqrt(np.prod(self.Kd))
 
 
+class gpuNUFFT:
+    """  GPU implementation of N-D non uniform Fast Fourrier Transform class.
+
+    Attributes
+    ----------
+    samples: np.ndarray
+        the normalized kspace location values in the Fourier domain.
```

Comment:
Can you also update in the `__init__` and specify normalized between what and what?

Reply:
But __init__ will not come up in the documentation .

Reply:
Added it for now still ..

Reply:
I think it depends which doc handler you are using. I Jupyter notebooks you have both.